### PR TITLE
a number of tmpfiles fixes

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: src/tmpfiles/tmpfiles.c [hunk 1]
```diff
@@ -784,49 +785,47 @@ static int path_set_perms(Item *i, const char *path) {
...
-                        if (m == (st.st_mode & 07777))
-                                log_debug("\"%s\" has right mode %o", path, st.st_mode);
-                        else {
-                                log_debug("chmod \"%s\" to mode %o", path, m);
-                                if (chmod(fn, m) < 0)
-                                        return log_error_errno(errno, "chmod() of %s via %s failed: %m", path, fn);
-                        }
+        if (S_ISLNK(st.st_mode))
+                log_debug("Skipping mode fix for symlink %s.", path);
+        else if (i->mode_set) {
```
Comment (yuwata) [inline src/tmpfiles/tmpfiles.c]: I prefer below here
```
if (i->mode_set) {
        if (S_ISLNK(st.st_mode))
                log_debug("Skipping mode fix for symlink %s.", path);
        else {
```
But this is just a debug message. No problem.

**Comment by yuwata** [comment]
This also fixes #7975.

File: src/tmpfiles/tmpfiles.c (hunk 1 above)
Comment (poettering) [reply src/tmpfiles/tmpfiles.c]: good point

**Comment by poettering** [comment]
Force pushed a new version that fixes what @yuwata pointed out. I also adding a tiny addition to UIDS-GIDS.md on top. And I referenced the new #7975 issue that popped up.

**Comment by yuwata** [comment]
lgtm.

**Pull request merged**